FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/DicomJobVersionManager.cs

```diff
@@ -0,0 +1,40 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Health.Fhir.Synapse.Core.Jobs.Models;
+
+namespace Microsoft.Health.Fhir.Synapse.Core.Jobs
+{
+    public class DicomJobVersionManager
```

Comment:
same here, rename to `DicomToDataLakeJobVersionManager`

Reply:
Updated, thanks.